exposed-java-time/src/main/kotlin/org/jetbrains/exposed/sql/javatime/JavaDateFunctions.kt

```diff
@@ -65,25 +63,31 @@ object CurrentDate : Function<LocalDate>(JavaLocalDateColumnType.INSTANCE) {
 }
 
 /**
- * Represents an SQL function that returns the current date and time, as [LocalDateTime].
+ * Represents an SQL function that returns the current date and time, as [LocalDateTime] with the specified fractional seconds [precision].
  *
  * @sample org.jetbrains.exposed.DefaultsTest.testConsistentSchemeWithFunctionAsDefaultExpression
  */
-object CurrentDateTime : CurrentTimestampBase<LocalDateTime>(JavaLocalDateTimeColumnType.INSTANCE)
+open class CurrentDateTime(precision: Byte? = null) : CurrentTimestampBase<LocalDateTime>(precision, JavaLocalDateTimeColumnType(precision)) {
+    companion object : CurrentDateTime()
```

Comment:
After changing `CurrentDateTime` from an object to a class, this line enables users to still use it like this `CurrentDateTime` and not be forced to use it like this `CurrentDateTime()`, thereby avoiding a breaking change.